front/lib/models/assistant/conversation.ts

```diff
@@ -464,11 +462,11 @@ Message.init(
     indexes: [
       {
         unique: true,
-        fields: ["conversationId", "rank", "version"],
+        fields: ["sId"],
```

Comment:
how is changing the order fixing it? I don't get it.

Reply:
see: https://github.com/dust-tt/tasks/issues/165#issuecomment-1786769114

Reply:
ok got it. Given the unexpected behavior of sequelize, I thought the order mattered.
Thanks.